pkg/bpfmetrics/metrics.go

```diff
@@ -0,0 +1,84 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright Authors of Tetragon
+package bpfmetrics
```

Comment:
I would make it a part of `pkg/metrics/eventmetrics`, is there a reason for having it as a separate package?

Reply:
No specific reason. I will move that to `pkg/metrics/eventmetrics`.